stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -1605,6 +1609,15 @@ public class ChatClient internal constructor(
             warmUp = false
         }
 
+        public fun customOkHttpClient(okHttpClient: OkHttpClient): Builder = apply {
+            this.customOkHttpClient = okHttpClient
+        }
```

Comment:
Seems to me like this API on its own would be enough for all use cases, and we don't need the second new method if we provide this one. I'd also name it `okHttpClient` to keep it simpler. Everything else around here looks all good to me!

Reply:
(The question about why we need both new methods is still open)

Reply:
Ops, I removed only the setter of the builder for httpClientConfig